Doc/library/enum.rst

```diff
@@ -804,7 +804,7 @@ Utilities and Decorators
 .. class:: auto
 
    *auto* can be used in place of a value.  If used, the *Enum* machinery will
-   call an *Enum*'s :meth:`_generate_next_value_` to get an appropriate value.
+   call an *Enum*'s :meth:`!Enum._generate_next_value_` to get an appropriate value.
```

Comment:
Why fix this, then unresolve it? If it didn't exist, it would be better to just silence it as-is, as now it just repeats the class name that was stated, in the possessive tense, in the very previous word? Moreover, this does exist, as it is an explicitly documented method that isn't mentioned before anywhere in this paragraph or information unit (or indeed, anywhere in this utility's description), so there's no reason not to just link it (with the class name hidden):

```suggestion
   call an *Enum*'s :meth:`~Enum._generate_next_value_` to get an appropriate value.
```

(As a sidenote, `` `*Enum* `` should ideally be changed to `` :class:`Enum` `` in the first instance and `` :class:`!Enum` `` in subsequent instances, but I didn't touch it here for consistency since there are a few other instances here, and I didn't want to go too out of scope here.)